### PR TITLE
Group components

### PR DESCRIPTION
**Pull request by blackinno**
- a reverse button group component

### CODE REVIEW

File: packages/Group/Button/ButtonGroup.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="btn-group">
+    <button v-for="(item,i) in items" :key="i" v-on:click="handleClicked(item.method,i)" :class="{actived: actived === i}">
+      <i v-if="item.prefix" :class="item.icon"></i>
```
Comment (antho1404) [inline packages/Group/Button/ButtonGroup.vue]: why not `pre-icon` and `post-icon` like that we avoid having a combination of 2 required properties.

**Comment by antho1404** [comment]
documentation please

**Comment by blackinno** [comment]
https://github.com/mesg-foundation/mesg-components/issues/70

File: packages/Choice/Choice.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import choiceRadio from './Radio'
+import choiceSelect from './Select'
+import choiceCheckbox from './Checkbox'
+
+export default {
+  name: 'Choice',
+  components: { choiceButton, choiceSelect, choiceRadio, choiceCheckbox },
+  props: {
+    type: { type: String, required: true },
+    value: { type: [String, Number], required: true },
```
Comment (antho1404) [inline packages/Choice/Choice.vue]: With checkboxes, we should have an array.

File: packages/Choice/Choice.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      },
+      set(value) {
+        return this.$emit('input', value)
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```
Comment (antho1404) [inline packages/Choice/Choice.vue]: to remove

**Comment by antho1404** [comment]
good for me, @EmmanuelDrouin can you check the design and merge if it's fine

**Review by antho1404 — APPROVED** [review]

**Comment by EmmanuelDrouin** [comment]
Looks good, only one thing about the "type button", on mobile each button should take full width (example: https://www.prisma.io/), otherwise it looks broken and will be hard to use.

For the other type looks good, i will work on the css later in another PR.

**Comment by EmmanuelDrouin** [comment]
Looks better, in the responsive version (mobile) you have an alignment problem because of the icon display before the content that actually should be in display none if we don't use it.
Screenshot 2019-11-15 at 11 58 06

You can see the text is not align in the middle.

If you use display none, the problem is fixed.
Screenshot 2019-11-15 at 12 01 36

Can you change that?

**Comment by EmmanuelDrouin** [comment]
Great let's merge it

**Pull request merged**